packages/sanity/src/desk/structureBuilder/util/getStructureBuilderId.ts

```diff
@@ -0,0 +1,16 @@
+import {camelCase} from 'lodash'
+import getSlug from 'speakingurl'
+import {disallowedPattern} from './validateId'
+
+export function getStructureBuilderId(title: string, id?: string): string {
```

Comment:
I suggest `getStructureNodeId()` - the "builder" part is kind of an implementation detail (would rename the file also to match, once the method is renamed).

```suggestion
export function getStructureNodeId(title: string, id?: string): string {
```